### PR TITLE
add option for tracking pxobjects

### PR DESCRIPTION
**Pull request by madanagopalt**

### CODE REVIEW

File: ci/build_linux.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ cd temp
 if [ "$TRAVIS_PULL_REQUEST" = "false" ]
 then
 	echo "************************* Generating config files *************************" >> $BUILDLOGS
-	cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON -DBUILD_DEBUG_METRICS=ON -DPXSCENE_TEST_HTTP_CACHE=ON .. >>$BUILDLOGS 2>&1;
+	cmake -DBUILD_PX_TESTS=ON -DBUILD_WITH_PXOBJECT_TRACKING=OFF  -DBUILD_PXSCENE_STATIC_LIB=ON -DBUILD_DEBUG_METRICS=ON -DPXSCENE_TEST_HTTP_CACHE=ON .. >>$BUILDLOGS 2>&1;
```
Comment (mfiess) [inline ci/build_linux.sh]: Are the changes in build_linux.sh needed?  BUILD_WITH_PXOBJECT_TRACKING currently defaults to OFF.

File: ci/build_osx.sh [hunk 2]
```diff
@@ -46,7 +46,7 @@ then
   echo "***************************** Generating config files ****" >> $BUILDLOGS
   if [ "$TRAVIS_EVENT_TYPE" != "cron" ] && [ "$TRAVIS_EVENT_TYPE" != "api" ] && [ -z "${TRAVIS_TAG}" ] 
   then
-    cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON -DBUILD_DEBUG_METRICS=ON -DPXSCENE_TEST_HTTP_CACHE=ON -DENABLE_THREAD_SANITIZER=ON .. >>$BUILDLOGS 2>&1;
+    cmake -DBUILD_PX_TESTS=ON -DBUILD_WITH_PXOBJECT_TRACKING=OFF -DBUILD_PXSCENE_STATIC_LIB=ON -DBUILD_DEBUG_METRICS=ON -DPXSCENE_TEST_HTTP_CACHE=ON -DENABLE_THREAD_SANITIZER=ON .. >>$BUILDLOGS 2>&1;
```
Comment (mfiess) [inline ci/build_osx.sh]: Are the changes in build_osx.sh needed? BUILD_WITH_PXOBJECT_TRACKING currently defaults to OFF.

File: examples/pxScene2d/src/CMakeLists.txt [hunk 3]
```diff
@@ -468,6 +469,10 @@ if (BUILD_WITH_TEXTURE_USAGE_MONITORING)
     set(PXSCENE_DEFINITIONS ${PXSCENE_DEFINITIONS} -DENABLE_PX_SCENE_TEXTURE_USAGE_MONITORING)
 endif (BUILD_WITH_TEXTURE_USAGE_MONITORING)
 
+if (BUILD_WITH_PXOBJECT_TRACKING)
```
Comment (mfiess) [inline examples/pxScene2d/src/CMakeLists.txt]: Let's add a message() statement here to indicate we are building with object tracking enabled.

Comment (madanagopalt) [reply examples/pxScene2d/src/CMakeLists.txt]: added message

**Pull request merged**